### PR TITLE
[14.0][FIX] web_edit_user_filter: Modified search facet for field type

### PR DESCRIPTION
**Pull request by anusriNPS**
To display user defined filter type in edit filter,including user defined filter values in description field of search facet element

### CODE REVIEW

**Comment by PicchiSeba** [comment]
Congratulations for your first PR!

Just a small fix regarding the PR structure: adapt the PR name to [_VERSION_][_TYPE_] **module_name**: short description
Other than that nice job. Let's wait for someone else to take a look at your changes

File: web_edit_user_filter/static/src/js/backend.js [hunk 1]
```diff
@@ -174,7 +187,10 @@ odoo.define("web_edit_user_filter/static/src/js/backend.js", function (require)
                     } else {
                         self.model.dispatch("toggleFilter", facetIdEl);
                     }
-                } else if (type === "groupBy") {
+                } else if (type === "field") {
+                    self.model.dispatch("toggleFilter", facetIdEl);
+                }
+                else if (type === "groupBy") {
                     self.model.dispatch("toggleFilter", facetIdEl);
```
Comment (PicchiSeba) [inline web_edit_user_filter/static/src/js/backend.js]: **suggestion**: We could merge these checks into a single one

```suggestion
                } else if (type === "groupBy" || type === "field") {
                    self.model.dispatch("toggleFilter", facetIdEl);
```

**Review by PicchiSeba — CHANGES_REQUESTED** [review]
Code Review: DRY

File: web_edit_user_filter/static/src/js/backend.js (hunk 1 above)
Comment (anusriNPS) [reply web_edit_user_filter/static/src/js/backend.js]: Done

**Review by PicchiSeba — APPROVED** [review]
Code review: LGTM

**Review by renda-dev — APPROVED** [review]
LGTM

**Comment by HekkiMelody** [comment]
@OCA/web-maintainers could this be merged? thanks

**Comment by simahawk** [comment]
@anusriNPS could you please rebase and push again to trigger a runboat build?

**Comment by anusriNPS** [comment]
@simahawk , rebased and verified changes with runboat instance

**Comment by simahawk** [comment]
@anusriNPS thanks. One last nit-picking thing... sorry, I didn't notice before: can you please remove the odoo version from the commit. See guidelines https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#commit-message :wink: 

Other than this, we are good to go!

**Comment by anusriNPS** [comment]
@simahawk , Updated commit message as per guidelines, please let me know if its fine

**Comment by simahawk** [comment]
Thanks!
/ocabot merge patch

**Pull request merged**